spyder/plugins/completion/confpage.py

```diff
@@ -15,7 +15,8 @@
 
 
 class CompletionConfigPage(PluginConfigPage):
-    def __init__(self, plugin, parent, providers=[]):
+    def __init__(self, plugin, parent, providers=None):
+        providers = [] if providers is None else providers
         super().__init__(plugin, parent)
```

Comment:
```suggestion
        super().__init__(plugin, parent)
        providers = [] if providers is None else providers
```

Our custom is not add code before calling `super` unless it's really necessary.